src/components/popup/popup.jsx

```diff
@@ -32,27 +18,24 @@ export default class Popup extends Component {
 	};
 
 	onCancel = () => {
-		this.setState({
```

Comment:
I would also reconsider modifying props at the expense of removing component's state. 
https://reactjs.org/docs/components-and-props.html#props-are-read-only

Reply:
Ya are right. I have rewritten code and not it uses `store.updateSection` function so `props` are not modified directly by component.